assets/components/src/category-autocomplete/style.scss

```diff
@@ -11,37 +11,23 @@
 		display: none;
 	}
 
-	.components-button:focus:not( :disabled ) {
-		box-shadow: inset 0 0 0 2px $primary-500;
-	}
-
-	// Multiple Select Controls
-
-	& + & {
-		margin-bottom: 32px;
-		margin-top: -16px;
-	}
-
-	// Multiple Controls
-
-	+ .newspack-select-control,
-	+ .newspack-text-control,
-	+ .newspack-textarea-control,
-	+ .newspack-toggle-control {
-		margin-bottom: 32px;
-		margin-top: -16px;
-	}
-
 	// Within a Grid
 
-	.newspack-grid & .newspack-form-token-field {
+	.newspack-grid & .newspack-form-token-field__input-container {
 		margin: 0;
 	}
 
 	// Loading state spinner
 	&__suggestions-spinner {
-		bottom: 0.5em;
```

Comment:
Not sure if this is desired, but the spinner is now appearing below the input:

<img width="374" alt="Screen Shot 2022-06-09 at 1 00 47 PM" src="https://user-images.githubusercontent.com/2230142/172924857-2fbf7e98-95a9-4749-b70f-a21558d88319.png">

Previously, it appeared inside the input:

<img width="360" alt="Screen Shot 2022-06-09 at 1 03 01 PM" src="https://user-images.githubusercontent.com/2230142/172924898-cce87dc8-eeca-4b3c-8fe8-65a10f75a459.png">

Reply:
Speaking of the spinner, it has a weird grey background and the positioning seems slightly off in Gutenberg editor pages. For example, try the Author List block, or an Author Profile block and search for an author name using the autocomplete field.

<img width="864" alt="Screen Shot 2022-06-09 at 1 06 47 PM" src="https://user-images.githubusercontent.com/2230142/172925665-b83cc46e-ba0c-4632-99ed-967daf459c35.png">
<img width="822" alt="Screen Shot 2022-06-09 at 1 09 24 PM" src="https://user-images.githubusercontent.com/2230142/172925670-07248ed6-9fb9-4998-8eea-5e619822de54.png">

Again, this only happens in Gutenberg editor pages. On the Components Demo page, the spinner looks fine in those contexts.

Reply:
Moving the spinner below the input is intentional but I might rework this.

I'm not sure what's happening with the author profile/list. I wonder if it's because of the new wp spinner component vs our newspack component. Might be worth testing again when we update newspack-components and update the npm package

Reply:
Spinner tweaked: https://github.com/Automattic/newspack-plugin/pull/1676/commits/89919197c2434145e8b003ee34d179fe0d609aa5